packages/terra-markdown/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Changelog
 Unreleased
 ----------
 ### Changed
+* Added prop for baseIUrl.
```

Comment:
typo: baseUrl

Reply:
And perhaps put these under `Added` and `Removed` sections instead of `Changed`.